_i18n/en/general/privacy.md

```diff
@@ -1,57 +1,82 @@
-AntennaPod is developed and maintained by individual volunteers and not represented by any legal entity. The community doesn't need your data, so the app and website are designed to be fully GDPR compliant. Read on to learn more.
+AntennaPod is developed and maintained by individual volunteers and is not represented by any legal entity. The community has no interest in your data, so the app and website are designed to be GDPR compliant. Read on to learn more.
```

Comment:
"has no interest" or "is not interested"? "Has no interest" sounds like something at a bank

Reply:
Both are grammatically fine:

> the feeling of wanting to give your attention to something or of wanting to be involved with and to discover more about something:
> *I've always had an interest in astronomy.*

https://dictionary.cambridge.org/dictionary/english/interest